content/blog/understanding-hpe-greenlake-identity-access-management.md

```diff
@@ -56,11 +58,11 @@ Roles are available for the services that are available within the tenant. The f
 | Billing Viewer                                  | View billing information                                                                                                                                                                                                  |
 | Billing Usage Viewer                            | View monthly charges card and report (usage information only - no cost information)                                                                                                                                       |
 |
-| **HPE GreenLake for Private Cloud**             |                                                                                                                                                                                                                           |
-| Private Cloud Tenant Owner                      | Administer HPE GreenLake for private cloud dashboard<br>Manage scheduling and activity<br>Manage infrastructure<br>Manage provisioning                                                                                    |
+| **HPE GreenLake for Private Cloud Enterprise: Virtual Machines**             |                                                                                                                                                                                                                           |
+| Private Cloud Tenant Owner                      | Administer HPE GreenLake for Private Cloud Enterprise: Virtual Machines dashboard<br>Manage scheduling and activity<br>Manage infrastructure<br>Manage provisioning                                                                                    |
 | Private Cloud Tenant Contributor                | Manage self-service VMs and app provisions                                                                                                                                                                                |
 |
-| **HPE GreenLake for Containers**                |                                                                                                                                                                                                                           |
+| **HPE GreenLake for Private Cloud: Containers on Virtual Machines**                |                                                                                                                                                                                                                           |
```

Comment:
Hi John,
Shouldn't it be "HPE GreenLake for Private Cloud Enterprise: Containers on Virtual Machines" ?